assert/assert_instance_of.ts

```diff
@@ -2,7 +2,7 @@
 import { AssertionError } from "./assertion_error.ts";
 
 // deno-lint-ignore no-explicit-any
-type AnyConstructor = new (...args: any[]) => any;
+export type AnyConstructor = new (...args: any[]) => any;
```

Comment:
Let's not export this, but copy it into expect.ts to avoid increasing the public types.